remote-access/web-server/nginx.md

```diff
@@ -106,7 +101,7 @@ sudo /etc/init.d/nginx reload
 Rename `index.html` to `index.php`:
 
 ```bash
-cd /usr/share/nginx/www/
+cd /var/www/html/
 sudo mv index.html index.php
```

Comment:
I believe that the file in /var/www/html is now called index.nginx-debian.html so this line should be updated as such.